src/OperationalInsights/OperationalInsights.Autorest/README.md

```diff
@@ -146,6 +146,13 @@ directive:
       parameter-name: Force
     set:
       parameter-name: ForceDelete
+	  
+   - where:
+      verb: Set
+      subject: AzOperationalInsightsWorkspace
+    set:
+      alias: Update-AzOperationalInsightsWorkspace
+	  
```

Comment:
```suggestion
	  
   - where:
      verb: Update
      subject: Workspace
    set:
      alias:
        - Set-AzOperationalInsightsWorkspace
        
```
The generated cmdlet named Update-AzOperationalInsightsWorkspace. We need keep align with handcrafted cmdlet Set-AzOperationalInsightsWorkspace, so we need add Set-AzOperationalInsightsWorkspace as the alias of Update-AzOperationalInsightsWorkspace 

For verb and subject:

   /// [OpenAPI] Update=>PATCH:"/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}"
    /// [DETAILS]
    /// verb: Update
    /// subjectPrefix: OperationalInsights
    /// subject: Workspace